idp/docker/config.yaml

```diff
@@ -15,6 +15,10 @@ TLS:
 Database:
   postgres:
     Port: 5432
+    MaxOpenConns: 200
+    MaxIdleConns: 20
+    MaxConnLifetime: 1800
+    MaxConnIdleTime: 1800
```

Comment:
These are all just best guesses at the moment.  We'll need to tune them over time.